react/week3/timeregistration/src/RegistrationDetails.js

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
what is this prop-types here and whats the use ?

Reply:
The homework stated that we must demostrate that we know how to use prototypes on components eg proptypes.shape, proptypes.strings, proptypes.number,.,.see "Technical details
All components should have prop types defined."